src/MathDistOps/pexp.py

```diff
@@ -0,0 +1,19 @@
+def pexp(x, rate, output_image=False):
+    """
+    Calculates the exponential distribution cumulative probability for a given value.
```

Comment:
I would also add that it gives you an image for the distribution in the description
```suggestion
    Calculates the exponential distribution cumulative probability for a given value and plots the corresponding distribution
```

Reply:
Sounds great! Thank you!